cypress.config.ts

```diff
@@ -67,5 +67,7 @@ export default defineConfig({
     excludeSpecPattern: '**/!(*.cy).ts',
     specPattern: 'integration_tests/tests/**/*.cy.{js,jsx,ts,tsx}',
     supportFile: 'integration_tests/support/index.ts',
+    viewportHeight: 800,
+    viewportWidth: 1020,
```

Comment:
I think it's worth noting in the commit message that this is the default maximum page width as specified here: https://design-system.service.gov.uk/styles/layout/